### ISSUE TITLE
Support Adding Additional Changelog Types

### ISSUE THREAD

**Issue by sam3k** [google-github-actions/release-please-action#838]
### TL;DR

I need to generate a changelog without PR links for each commit and would like to do so by adding my own `changelog-type` as described [here](https://github.com/googleapis/release-please/blob/main/docs/customizing.md#adding-additional-changelog-types).

### Detailed design

Docs mentioned that we can use [ChangelogNotes interface](https://github.com/googleapis/release-please/blob/main/src/changelog-notes.ts) to add [additional changelog types](https://github.com/googleapis/release-please/blob/main/docs/customizing.md#adding-additional-changelog-types); but this seems to not apply to the Github action.

**Comment by chingor13** [comment]
See related: https://github.com/googleapis/release-please/issues/2080

**Comment by danielo515** [comment]
It will be nice if they provide some clear and concise instructions about how to do things. As by most google products, the existing documentation is ambiguous and it is not clear if some things are even possible.
